spec/models/manageiq/providers/amazon/storage_manager/ebs/stubbed_refresher_spec.rb

```diff
@@ -82,6 +82,7 @@ def refresh_spec_full
 
     assert_specific_snapshot
     assert_specific_volume
+    assert_unencrypted_snapshot
```

Comment:
Change the order - move assert_unencrypted_snapsthot before assert_specific_volume.